### ISSUE TITLE
svg example fails on Raspbian Stretch Lite (object has no attribute '_filename')

### ISSUE THREAD

**Issue by OutsourcedGuru** [kivy/kivy#6110]
### Versions

* Python: v2.7.13
* OS: Raspbian Stretch Lite from 4/18/2018 release: "Linux raspberrypi 4.14.79-v7+ #1159 SMP Sun Nov 4 17:50:20 GMT 2018 armv7l GNU/Linux"
* Kivy: v1.11.0.dev0
* Kivy installation method:

```
* `sudo apt-get update`
* `sudo apt-get -y upgrade`
* `sudo apt-get install libsdl2-dev libsdl2-image-dev libsdl2-mixer-dev libsdl2-ttf-dev pkg-config libgl1-mesa-dev libgles2-mesa-dev python-setuptools libgstreamer1.0-dev git-core gstreamer1.0-plugins-{bad,base,good,ugly} gstreamer1.0-{omx,alsa} python-dev libmtdev-dev xclip xsel`
* `source oprint/bin/activate`
* `pip install -U Cython==0.28.2`
* `cd ~`
* `git clone https://github.com/kivy/kivy`
* `cd kivy && make` # Requires activating the venv first
* `echo "export PYTHONPATH=$(pwd):\$PYTHONPATH" >> ~/.bashrc`
* `echo "export KIVY_GL_BACKEND=gl" >> ~/.bashrc`
* `source ~/.bashrc`
* `deactivate`
```

### Description

I wanted to run the svg examples, seeing them on an attached 10" display which has been working well otherwise.

The examples in this folder fail. There seems to be a (new) expectation perhaps for the underlying `kivy/graphics/svg.pyx` file that the "_filename" be an instance but I'm not sure. It's possible that `svg.pyx` was changed and yet nobody tested/changed the example code.

### Code and Logs

```
~/kivy/examples/svg$ python main.py
[INFO   ] [Logger      ] Record log in /home/pi/.kivy/logs/kivy_19-01-24_20.txt
[INFO   ] [Kivy        ] v1.11.0.dev0, git-372b819, 20181227
[INFO   ] [Python      ] v2.7.13 (default, Nov 24 2017, 17:33:09) 
[GCC 6.3.0 20170516]
[INFO   ] [Factory     ] 184 symbols loaded
[INFO   ] [Image       ] Providers: img_tex, img_dds, img_sdl2, img_gif (img_pil, img_ffpyplayer ignored)
[INFO   ] [Window      ] Provider: egl_rpi
[INFO   ] [GL          ] Using the "OpenGL ES 2" graphics system
[INFO   ] [GL          ] Backend used <gl>
[INFO   ] [GL          ] OpenGL version <OpenGL ES 2.0>
[INFO   ] [GL          ] OpenGL vendor <Broadcom>
[INFO   ] [GL          ] OpenGL renderer <VideoCore IV HW>
[INFO   ] [GL          ] OpenGL parsed version: 2, 0
[INFO   ] [GL          ] Shading version <OpenGL ES GLSL ES 1.00>
[INFO   ] [GL          ] Texture max size <2048>
[INFO   ] [GL          ] Texture max units <8>
[INFO   ] [Window      ] virtual keyboard allowed, single mode, docked
 Traceback (most recent call last):
   File "main.py", line 50, in <module>
     SvgApp().run()
   File "/home/pi/kivy/kivy/app.py", line 800, in run
     root = self.build()
   File "main.py", line 43, in build
     svg = SvgWidget(filename, size_hint=(None, None))
   File "main.py", line 29, in __init__
     svg = Svg(filename)
   File "kivy/graphics/svg.pyx", line 419, in kivy.graphics.svg.Svg.__init__
 AttributeError: 'kivy.graphics.svg.Svg' object has no attribute '_filename'
```

Unaltered `/home/pi/kivy/examples/svg/main.py` contents:

```
import sys
from glob import glob
from os.path import join, dirname
from kivy.uix.scatter import Scatter
from kivy.app import App
from kivy.graphics.svg import Svg
from kivy.core.window import Window
from kivy.uix.floatlayout import FloatLayout
from kivy.lang import Builder

Builder.load_string("""
<SvgWidget>:
    do_rotation: False
<FloatLayout>:
    canvas.before:
        Color:
            rgb: (1, 1, 1)
        Rectangle:
            pos: self.pos
            size: self.size
""")


class SvgWidget(Scatter):

    def __init__(self, filename, **kwargs):
        super(SvgWidget, self).__init__(**kwargs)
        with self.canvas:
            svg = Svg(filename)
        self.size = svg.width, svg.height


class SvgApp(App):

    def build(self):
        self.root = FloatLayout()

        filenames = sys.argv[1:]
        if not filenames:
            filenames = glob(join(dirname(__file__), '*.svg'))

        for filename in filenames:
            svg = SvgWidget(filename, size_hint=(None, None))
            self.root.add_widget(svg)
            svg.scale = 5.
            svg.center = Window.center

if __name__ == '__main__':
    SvgApp().run()
```

**Comment by OutsourcedGuru** [comment]
This appears to be fixed by [this](https://github.com/kivy/kivy/commit/5620413756ee1ea835845846bdbad62cbbb029bc).

Thanks.

By the way, the `tiger.svg` in that `kivy/examples/svg` folder causes a segmentation fault on Raspbian even with the various fixes like...

```
import os
os.environ['KIVY_GL_BACKEND'] = 'gl'
import kivy
```

I would suggest nuking that from the examples.

**Comment by tito** [comment]
Fixed in #6112
